tests/legacy-cli/e2e/tests/misc/completion.ts

```diff
@@ -0,0 +1,51 @@
+import { execAndWaitForOutputToMatch } from '../../utils/process';
+
+export default async function () {
+  // ng build
+  await execAndWaitForOutputToMatch('ng', ['--get-yargs-completions', 'b', ''], /test-project/);
+  await execAndWaitForOutputToMatch('ng', ['--get-yargs-completions', 'build', ''], /test-project/);
+  await execAndWaitForOutputToMatch('ng', ['--get-yargs-completions', 'build', '-a'], /aot/);
```

Comment:
Does this autocomplete to `-aot` or `--aot`?

Reply:
`--aot`